src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -162,28 +171,6 @@ private Task<IEnumerable<ISymbol>> FindAsyncWorker(SearchQuery query, AsyncLazy<
             return result;
         }
 
-        /// <summary>
-        /// Slow, linear scan of all the symbols in this assembly to look for matches.
-        /// </summary>
-        private async Task<IEnumerable<ISymbol>> FindAsync(AsyncLazy<IAssemblySymbol> lazyAssembly, Func<string, bool> predicate, CancellationToken cancellationToken)
```

Comment:
which means we can remove this.  which also means it's easier to make a SymboltreeInfo that stores data more efficiently.